### ISSUE TITLE
Does Ditto supports HA in K8s?

### ISSUE THREAD

**Issue by tiexinyang** [eclipse/ditto#711]
I'm using ditto 1.1.0 in K8s cluster, it works well while using single pod for each component.
But when I try to extend component Things & Policies with multi replicas, pods will not able to join the ditto-cluster and show some wrong status.

> [XXX ditto-1.1.0]$ kubectl get pods
> NAME                             READY   STATUS             RESTARTS   AGE
> concierge-6b79df446b-vpz2w       0/1     Running            2          25m
> connectivity-5cc955ddcb-x6r9g    0/1     Running            1          25m
> gateway-5d9fc78f5b-pv699         1/1     Running            2          25m
> gateway-5d9fc78f5b-vbglp         1/1     Running            2          25m
> nginx-c7447b4bc-sq29p            1/1     Running            0          25m
> nginx-c7447b4bc-tz7j7            1/1     Running            0          25m
> policies-5b8b479546-dxlnw        0/1     Running            2          22m
> policies-5b8b479546-fs8jz        0/1     Running            2          25m
> policies-5b8b479546-fvn9r        0/1     Running            3          22m
> policies-5b8b479546-zczgj        0/1     CrashLoopBackOff   1          22m
> swagger-ui-9588fd54b-vkztc       1/1     Running            0          25m
> things-5cf78667d-n99bd           0/1     CrashLoopBackOff   1          17m
> things-5cf78667d-q7j4c           0/1     Running            2          25m
> things-5cf78667d-x56nz           0/1     Running            3          17m
> things-5cf78667d-zbjqz           0/1     Running            2          17m
> things-search-757c9b74fb-nzlhl   0/1     Running            1          25m

Does it mean that ditto cannot support HA in K8s? Or is there any proper solutions?

**Comment by stmaute** [comment]
Hi @tiexinyang 
are you sure you have enough cpu and memory to run multiple instances of things and policies service?
Could you share the kubernetes logs of these pods and if available the log of things or policies service?

**Comment by tiexinyang** [comment]
> Hi @tiexinyang
> are you sure you have enough cpu and memory to run multiple instances of things and policies service?
> Could you share the kubernetes logs of these pods and if available the log of things or policies service?

Hi,
Thanks for your notification.
I checked the log and found there is some delay on the heartbeat, which cause the pods dropped by the ditto cluster from time to time, that's because I set the resource limits for both CPU & Memory in case of the OOM, now I removed the limits and found everything runs well.
Ditto is now able to support HA smoothly.
